Sources/SwiftSyntax/Raw/RawSyntaxTokenView.swift

```diff
@@ -197,6 +197,28 @@ public struct RawSyntaxTokenView {
     }
   }
 
+  /// Returns a `RawSyntax` node with the presence changed to `newValue`.
+  @_spi(RawSyntax)
+  public func withPresence(_ newValue: SourcePresence, arena: SyntaxArena) -> RawSyntax {
+    switch raw.rawData.payload {
+    case .parsedToken(_):
```

Comment:
Is there a reason to keep `(_)`?

Reply:
Technically, there is not. But it matches the style in the rest of the file.